data/changelog.txt

```diff
@@ -131,6 +150,12 @@ Hound afterimages also copy their host nicknames
 The player is substantially less effective with guns at low skill values
 Lycra is less protective
 BMI has a less all consuming impact on how healthy you are
+you can't uninstall bio_int_enhancer or bio_memory
+Being badly wounded will always allow you to swap characters in camp while you heal
+microlab reactor has nuclear fuel cells instead of asrgs
+certain previously non dodgeable attacks can be dodged
+intelligence provides a multiplier to current focus rather than adding to its value
+titanium skeletal bracing provides bash armor
```

Comment:
```suggestion
Being badly wounded will always allow you to swap characters in camp while you heal
Certain previously non dodgeable attacks can be dodged
Intelligence provides a multiplier to current focus rather than adding to its value
```